content/blog/getting-started-with-automatic-releases/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Getting started with automatic releases'
```

Comment:
I'd like to change the title to "5 Steps to automatic releases" or something that has the number in the title.